src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXOverrideMismatch/PXOverrideMethodSymbolComparer.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using System.Linq;
+using Acuminator.Utilities.Common;
+
+namespace Acuminator.Analyzers.StaticAnalysis.PXOverrideMismatch
+{
+	public class PXOverrideMethodSymbolComparer : IEqualityComparer<IMethodSymbol>
+	{
+		public static PXOverrideMethodSymbolComparer Instance = new PXOverrideMethodSymbolComparer();
+
+		public bool Equals(IMethodSymbol src, IMethodSymbol ext)
+		{
+			var srcParams = src.Parameters.Select(p => p.Type).ToImmutableArray();
+			var extParams = ext.Parameters.Select(p => p.Type).ToImmutableArray();
+
+			var isSimple = srcParams.Length == extParams.Length;
```

Comment:
Please use more sensible names. It's not good that Acumatica PXOverride resolution logic have not sensible names but we have a stricter code quality policy for Acuminator.

Reply:
Names are reworked now. Thank you for your help.